docs/running/source.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Building Conduit from Source'
+sidebar_position: 3
+hide_title: true
+sidebar_label: "Build from Source"
+---
+
+# Building from source
+
+Before you can build Conduit from source, you need to have Go 1.18 or higher installed. The Go website has [comprehensive instructions](https://go.dev/doc/install) on how to get the language installed on your machine. Once installed, you can follow the rest of the instructions.
```

Comment:
Go 1.21 is what we require now: https://github.com/ConduitIO/conduit?tab=readme-ov-file#build-from-source